cookbooks/fb_systemd/recipes/udevd.rb

```diff
@@ -18,19 +18,26 @@
 # limitations under the License.
 #
 
+udevadm = value_for_platform(
+  'centos' => {
+    '< 6.0' => '/sbin/udevadm'
```

Comment:
Add a trailing comma here for consistency

Reply:
> Add a trailing comma here for consistency

Fixed.